JsonFormatter/FormatterJson.cs

```diff
@@ -34,7 +34,7 @@ private void GetJson(TreeNode<T> tree, int level, bool isRoot,StringBuilder resu
                 {
                     countSpaces.Append(" ");
                 }
-                PropertyInfo[] props = tree.Data.GetType()
+                var props = tree.Data.GetType()
                     .GetProperties(BindingFlags.Public | BindingFlags.Instance);
```

Comment:
What such flags like "public" or "Instance" means? Why they are needed here?

Reply:
When we work with bindings flags in method GetProperties to get a return value we have to use BindingFlags.Instance or BindingFlags.Static. In this case i want to get instance properites which have public access modifier.

Reply:
If we wouldn't use BindingFlags.Public we wouldn't have access to properties which have public access modifier? What will happened if we remove flags and use pure .GetProperties() method? Will it work or not? Why?

Reply:
It will work . GetProperties() without flag realize in his body method GetProperties(Type.DefaultLookup) . Type.DefaultLookup includes binding flags:instance, static, public.